src/components/Client/ClientLogin.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import axios from 'axios';
+
+const ClientLogin = () => {
+    const history = useHistory();
+    const [phone, setPhone] = useState('')
+    const [password, setPassword] = useState('')
```

Comment:
combine password related state props in one obj, like `
{
  password: '', 
  passwordAlert: ''
}
`